front-end/app/components/map-comp.hbs

```diff
@@ -0,0 +1,20 @@
+<LeafletMap
+  @lat={{this.lat}}
+  @lng={{this.lng}}
+  @zoom={{this.zoom}}
+  @onClick={{this.onMapClick}}
+  as |layers|
+>
+  <layers.tile
+    @url='https://{s}.basemaps.cartocdn.com/light_all/{z}/{x}/{y}.png'
+  />
+  <layers.marker @location={{this.location}} as |markerLayer|>
+    <markerLayer.popup>
+      <h3>The Oregon Convention Center</h3>
+      777 NE Martin Luther King Jr Blvd<br />
+      Portland, OR 97232
+    </markerLayer.popup>
+  </layers.marker>
+</LeafletMap>
+
+{{outlet}}
```

Comment:
The outlet tag is used so that it's display on every page. It's kinda an global override, so I'd suggest removing it, otherwise it will show up on every page.
```suggestion

```